specs/dvc/add_file_to_remote_storage.yml

```diff
@@ -0,0 +1,32 @@
+---
+name: Add file to remote storage and/or an upstream repo
```

Comment:
This looks awesome. Can we indicate that this workflow is related to dvc in the workflow name?